### PR TITLE
Separate into two workflows the release drafter and the release notes jobs

### PR DESCRIPTION
**Pull request by ramrodo**
Separate into two workflows these jobs:
- `release-drafter`: creates a draft PR including every change merged into master
- `release-notes`: writes down in a markdown file the changes for the next release and creates a PR using this markdown file. This workflow will be triggered **only** on creating `v*` tags.

Signed-off-by: Rodolfo Martínez Vega <rodomar@outlook.com>

**Description**

This PR fixes #2612

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -11,20 +11,20 @@ categories:
       - 'kind/bugfix'
       - 'kind/bug'
   - title: '🧰 Maintenance'
-    labels: 
+    labels:
```
Comment (leecalcote) [inline .github/release-drafter.yml:14]: ✔️

File: .github/workflows/release-drafter.yml [hunk 2]
```diff
@@ -3,9 +3,7 @@ name: Release Drafter
 on:
   push:
     branches:
-      - '!*'
-    tags:
-      - 'v*'
+      - master
```
Comment (leecalcote) [inline .github/workflows/release-drafter.yml:6]: ✔️

File: .github/workflows/release-notes.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+name: Release notes
```
Comment (leecalcote) [inline .github/workflows/release-notes.yml:1]:
```suggestion
name: Release Notes Publisher
```

File: .github/workflows/release-notes.yml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  create_release_draft:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Drafting release
+          id: release_drafter
+          uses: release-drafter/release-drafter@v5
+          with:
+            config-name: release-drafter.yml
+          env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (leecalcote) [inline .github/workflows/release-notes.yml:20]: Not sure if `secrets.RELEASE_NOTES_PAT` will be needed or not. We'll see...

**Review by leecalcote — DISMISSED** [review]
Nice!

File: .github/workflows/release-notes.yml (hunk 4 above)
Comment (ramrodo) [reply .github/workflows/release-notes.yml:20]: Ok

**Pull request merged**